### PR TITLE
Pixman Update to 0.40.0, use clang linker

### PR DESCRIPTION
**Pull request by satmandu**
Part of https://github.com/skycocker/chromebrew/pull/4450

Works properly:
- [x] x86_64

### CODE REVIEW

File: packages/pixman.rb [hunk 1]
```diff
@@ -3,25 +3,16 @@
...
-    aarch64: '056ddb91ba2761a2c742fe47edb02c9a2dd2a2ae89ff06b5a4733d346ee488ff',
-     armv7l: '056ddb91ba2761a2c742fe47edb02c9a2dd2a2ae89ff06b5a4733d346ee488ff',
-       i686: '07e84363add2a3e345340243af5fe39dc7c2b1d5fd3d5252bb64080b86abf12c',
-     x86_64: 'c268902b79b7dffe7214fb0f561e9ecdb9be47309d2440da3a386cabe3ba4370',
-  })
+  depends_on 'llvm' => ':build'
 
   def self.build
+    ENV['CFLAGS'] = "-fuse-ld=lld"
+    ENV['CXXFLAGS'] = "-fuse-ld=lld"
```
Comment (uberhacker) [inline packages/pixman.rb:15]: Does this break arm builds?

Comment (satmandu) [reply packages/pixman.rb:15]: I'll wait until some version of https://github.com/skycocker/chromebrew/pull/4483 is merged to figure out if there are issues?

The armv7l image I am using from https://gist.github.com/satmandu/791dd6e982d365dda6bbba5c332f0f73 hasn't been complaining when it sees armv7l as the ARCH and #{CREW_OPTONS} is set for the apps I have tested so far.

File: packages/pixman.rb [hunk 2]
```diff
@@ -3,25 +3,16 @@
...
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
```
Comment (uberhacker) [inline packages/pixman.rb:16]: Does `#{CREW_OPTIONS}` work?

File: packages/pixman.rb (hunk 1 above)
Comment (satmandu) [reply packages/pixman.rb:15]: gcc10 should fix these issues.

File: packages/pixman.rb (hunk 2 above)
Comment (satmandu) [reply packages/pixman.rb]: yup

**Pull request merged**